doc/tutorials/scripts/scripting.md

```diff
@@ -9,6 +9,11 @@ Playnite can be extended with additional functionality using scripts. [PowerShel
 > [!NOTE] 
 > PowerShell support requires at least PowerShell 5.0 to be installed. If you are Windows 7 user you need to [install it manually](https://www.microsoft.com/en-us/download/details.aspx?id=54616) (Windows 8 and 10 includes it by default).
 
+PowerShell extensions are imported as a [PowerShell module](https://docs.microsoft.com/en-us/powershell/scripting/developer/module/how-to-write-a-powershell-script-module?view=powershell-5.1). The extension of the file must be `.psm1` (or `.psd1` if you use a [PowerShell module manifest](https://docs.microsoft.com/en-us/powershell/scripting/developer/module/how-to-write-a-powershell-module-manifest?view=powershell-5.1)). Any exported functions from your extension must be exported from the module. In a `.psm1` file all functions in the module scope are exported by default, but functions that are in the global scope (defined like `function global:OnGameStarted()`) will _not_ be correctly exported. Exported functions must accept the _exact_ number of arguments that Playnite passes to them.
+
+> [!NOTE]
+> Importing PowerShell extensions as modules is a breaking change as of Playnite 9. Existing extensions will be need to be updated. For many extensions this will be as simple as renaming the file from `.ps1` to `.psm1`, updating the manifest and removing `global:` from any exported functions. A `params()` section may need to be added to exported functions that were not explicitly accepting the correct number of arguments.

```

Comment:
I think this and the note above it are enough to explain the new changes about importing PowerShell scripts as modules. Please let me know if this needs to be clarified or expanded anywhere.

Reply:
I think it would be better to have it in the changelog and create separate page that explains how to migrate from Playnite 8 to 9. People just starting with scripting won't care about breaking changes from earlier versions.

Reply:
Ok, I added the note as a **Breaking Change** in the changelog. I put it in a new "TODO: 5.x.x" section, you can update it when you make a release. I left it as paragraph text, not bullet points. If you want it restructured we can change it to bullet points.